raster/src/main/scala/geotrellis/raster/merge/TileMergeMethods.scala

```diff
@@ -25,7 +25,7 @@ import geotrellis.vector.Extent
 /**
   * Trait guaranteeing extension methods for doing merge operations on [[Tile]]s.
   */
-trait TileMergeMethods[T] extends MethodExtensions[T] {
+abstract class TileMergeMethods[T <: CellGrid[Int]] extends MethodExtensions[T] {
```

Comment:
Was the `CellGrid[Int]` constraint here needed?

Reply:
Nope, not sure why I did that.